### ISSUE TITLE
SUBCORE not defined for Signal Processing examples

### ISSUE THREAD

**Issue by robotastic** [sonydevworld/spresense-arduino-compatible#168]
I am trying to use the Peak and Sound Detector Examples. The Main Core code compiles correctly. When I try to compile the SubCore examples I get a similar error:

````
/private/var/folders/7k/qtt4sc3j2sq1rx4yc452d38r0000gn/T/.arduinoIDE-unsaved2022513-22396-161duwj.eqik/SubFFT/SubFFT.ino: In function 'void setup()':
/private/var/folders/7k/qtt4sc3j2sq1rx4yc452d38r0000gn/T/.arduinoIDE-unsaved2022513-22396-161duwj.eqik/SubFFT/SubFFT.ino:76:22: error: no matching function for call to 'MPClass::begin()'
   76 |   int ret = MP.begin();
      |                      ^
In file included from /private/var/folders/7k/qtt4sc3j2sq1rx4yc452d38r0000gn/T/.arduinoIDE-unsaved2022513-22396-161duwj.eqik/SubFFT/SubFFT.ino:20:
/Users/luke/Library/Arduino15/packages/SPRESENSE/hardware/spresense/2.5.1/libraries/MP/src/MP.h:127:7: note: candidate: 'int MPClass::begin(int)'
  127 |   int begin(int subid);
      |       ^~~~~
/Users/luke/Library/Arduino15/packages/SPRESENSE/hardware/spresense/2.5.1/libraries/MP/src/MP.h:127:7: note:   candidate expects 1 argument, 0 provided
/private/var/folders/7k/qtt4sc3j2sq1rx4yc452d38r0000gn/T/.arduinoIDE-unsaved2022513-22396-161duwj.eqik/SubFFT/SubFFT.ino: In function 'void loop()':
/private/var/folders/7k/qtt4sc3j2sq1rx4yc452d38r0000gn/T/.arduinoIDE-unsaved2022513-22396-161duwj.eqik/SubFFT/SubFFT.ino:100:33: error: no matching function for call to 'MPClass::Recv(int8_t*, Request**)'
  100 |   ret = MP.Recv(&rcvid, &request);
      |                                 ^
In file included from /private/var/folders/7k/qtt4sc3j2sq1rx4yc452d38r0000gn/T/.arduinoIDE-unsaved2022513-22396-161duwj.eqik/SubFFT/SubFFT.ino:20:
/Users/luke/Library/Arduino15/packages/SPRESENSE/hardware/spresense/2.5.1/libraries/MP/src/MP.h:176:7: note: candidate: 'int MPClass::Recv(int8_t*, uint32_t*, int)'
  176 |   int Recv(int8_t *msgid, uint32_t *msgdata, int subid);
      |       ^~~~
/Users/luke/Library/Arduino15/packages/SPRESENSE/hardware/spresense/2.5.1/libraries/MP/src/MP.h:176:7: note:   candidate expects 3 arguments, 2 provided
/Users/luke/Library/Arduino15/packages/SPRESENSE/hardware/spresense/2.5.1/libraries/MP/src/MP.h:210:7: note: candidate: 'int MPClass::Recv(int8_t*, void*, int)'
  210 |   int Recv(int8_t *msgid, void *msgaddr, int subid);
      |       ^~~~
/Users/luke/Library/Arduino15/packages/SPRESENSE/hardware/spresense/2.5.1/libraries/MP/src/MP.h:210:7: note:   candidate expects 3 arguments, 2 provided

Compilation error: exit status 1
````

It looks like SUBCORE may not being defined correctly somewhere based on this lines from the MP Library:
https://github.com/sonydevworld/spresense-arduino-compatible/blob/8b249bac8330070489b088ee9224d862a034f2b7/Arduino15/packages/SPRESENSE/hardware/spresense/1.0.0/libraries/MP/src/MP.h#L124-L128

I am using an M1 MacBook Pro running Arduino IDE Version: 2.0.0-rc7

**Comment by SPRESENSE** [comment]
Hi @robotastic san

When compiling & uploading a SubFFT sketch, please select `SubCore 1` for `Core` from Arduino IDE menu.

![SubFFT-upload](https://user-images.githubusercontent.com/41312067/173726818-09fbf476-d1eb-490b-b491-4bba127c8d62.png)

Please see [Tutorial of SignalProcessing Library](https://developer.sony.com/develop/spresense/docs/arduino_tutorials_en.html#_operating_procedure_12) for more details.

**Comment by SPRESENSE** [comment]
Fixed and released in v2.6.0.
